LBHackney-IT/lbhackney-it.github.io
# Add GitHub management runbooks

In order to formalise the Joiners/Movers/Leavers process, and to make it obvious how to do things in GitHub, we need runbooks to detail how we add and remove users. This change adds some 🙂.

### Motivation

We now have a more complex GitHub setup, with SAML SSO for login. This means we have to add users in a different way to the obvious route in the UI (and docs). 

### What this change does

This PR adds a few pages to the Ways of Working section which detail:

- How to add a user to GitHub
- How to remove a user from GitHub
- How we structure teams in GitHub
- Who owns a GitHub user account

It also adds some sidebar position metadata to surrounding pages to preserve an intuitive ordering in the sidebar.

There's probably more we could add, but these are the things I've talked about most as we enabled SAML SSO.

### How to test

The content and links should all read and render well both in the static site and in GitHub. This is a content change so proof-reading is the most valuable.

Rich diff is really handy to check quotes/info bits render as you'd expect, or just to make it easier to read:
![Screenshot 2024-11-04 at 16 16 (Safari)@2x](https://github.com/user-attachments/assets/0ae7b6cc-8fc1-4d8a-9485-5ccefd5e9aab)

### Screenshot

![Screenshot 2024-11-04 at 16 14 (Brave Browser)@2x](https://github.com/user-attachments/assets/bf8564d9-d040-45c7-af21-3682551db449)
